### PR TITLE
Update scripts and docs for 1.1.6

### LINKED ISSUE #2196: Update changelog and scripts for `1.1.6`

### LINKED ISSUE #2214: Update install docs to direct users to the marketplace (via the extension sidebar)

### PR DESCRIPTION
**Pull request by dotNomad**
This PR preps us for the release of `1.1.6` by:
- Updating the scripts to point to `1.1.6`
- Adds details for the release into the changelog
- Adds additional licenses
- Updates our VS Code Extension documentation

## Intent

Resolves #2196
Resolves #2214

## Type of Change

- - [ ] Bug Fix 
- - [ ] New Feature 
- - [ ] Breaking Change 
- - [x] Documentation 
- - [ ] Refactor 
- - [ ] Tooling

### CODE REVIEW

**Comment by dotNomad** [comment]
Once #2189 is merged this will need another license update. Marking as draft until that is complete.

**Comment by dotNomad** [comment]
Updated off of `main` and ready for review

File: docs/installation.md [hunk 1]
```diff
@@ -7,11 +7,11 @@ VSCode.
 
 Download and install the VSCode extension.
 
-- For Arm MacOS: [publisher-1.1.5-darwin-arm64.vsix](https://cdn.posit.co/publisher/releases/tags/v1.1.5/publisher-1.1.5-darwin-arm64.vsix)
-- For Intel MacOS: [publisher-1.1.5-darwin-amd64.vsix](https://cdn.posit.co/publisher/releases/tags/v1.1.5/publisher-1.1.5-darwin-amd64.vsix)
-- For Windows: [publisher-1.1.5-windows-amd64.vsix](https://cdn.posit.co/publisher/releases/tags/v1.1.5/publisher-1.1.5-windows-amd64.vsix)
-- For Arm Linux: [publisher-1.1.5-linux-arm64.vsix](https://cdn.posit.co/publisher/releases/tags/v1.1.5/publisher-1.1.5-linux-arm64.vsix)
-- For Intel Linux: [publisher-1.1.5-linux-amd64.vsix](https://cdn.posit.co/publisher/releases/tags/v1.1.5/publisher-1.1.5-linux-amd64.vsix)
+- For Arm MacOS: [publisher-1.1.6-darwin-arm64.vsix](https://cdn.posit.co/publisher/releases/tags/v1.1.6/publisher-1.1.6-darwin-arm64.vsix)
```
Comment (sagerb) [inline docs/installation.md:10]: Shouldn't we just update this now to download from the marketplace?

**Review by sagerb — COMMENTED** [review]
I think this would be a good time to move away from the vsix installation instructions...

File: docs/installation.md (hunk 1 above)
Comment (dotNomad) [reply docs/installation.md:14]: I added instructions and a link to the marketplace page for installation. I kept the manual installation instructions to keep this PR from becoming destructive, but I can absolutely remove them.

**Review by sagerb — APPROVED** [review]
This is good for now. Thanks!

**Pull request merged**